Sustainability/OCP-Sustainability-Profile.md

```diff
@@ -0,0 +1,489 @@
+
+To produce this guide, DMTF's [Redfish Documentation Generator](#redfish-documentation-generator) merges DMTF's Redfish Schema bundle (DSP8010) contents with supplemental text.
+
+## <a name="using-the-reference-guide"></a>Using the reference guide
+
+Every Redfish response consists of a JSON payload containing properties that are strictly defined by a schema for that Resource.  The schema defining a particular Resource can be determined from the value of the "@odata.type" property returned in every Redfish response.  This guide details the definitions for every Redfish standard schema.
+
+Each schema section contains:
+
+* The schema's name, its current version, and description.
+* The schema release history, which lists each minor schema version and the DSP8010 release bundle that includes it.
+* The list of URIs where schema-defined Resources appear in a Redfish Service v1.6 and later.  For more information, see [URI listings](#uri-listings).
+* The table of properties, which includes additional property details, when available.
+* The list of available schema-defined actions.
+* The example schema-defined JSON payload for a Resource.
+
+<br>
+The property-level details include:
+
+| Column | Purpose |
+| :--- | :--------- |
+| Property name | <p>The case-sensitive name of the JSON property as it appears in the JSON payload.</p><p>Lists the schema version in parentheses when properties were added to or deprecated in the schema after the initial v1.0.0 release.</p> |
+| Requirements | <p>The property-level read and write requirements as listed in the Redfish Profile.</p> |
+| Type | <p>The JSON data types for the property, which can include boolean, number, string, or object.</p><p>The <code>string (enum)</code> tag identifies enumerated strings.</p><p>Number types that use units specify the units.</p> |
+| Description | <p>The normative description of the property, as copied directly from the schema <code>LongDescription</code> definition.</p> |
+
+
+## <a name="chassis-1.24.0-%28general%29"></a>Chassis 1.24.0 (General)
+
+### Description
+
+This section describes a UseCase of Chassis.
+
+A service is required to implement this UseCase. (Mandatory)
+
+This UseCase may be found at the following URIs: 
+
+### URIs
+
+/&#8203;redfish/&#8203;v1/&#8203;Chassis/&#8203;*{ChassisId}*<br>
+
+
+### Properties
+
+|Property     |Type     |Attributes   |Notes     |
+| :--- | :--- | :--- | :--------------------- |
+| **ChassisType** | string<br>(enum) | *Mandatory (Read-only)* | The type of physical form factor of the chassis. *For the possible property values, see ChassisType in Property details.* |
+| **EnvironmentMetrics** *(v1.15+)* { | object | *Supported (Read-only)* | The link to the environment metrics for this chassis. |
+| &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;**@odata.context** | string<br>(URI) | *Mandatory (Read-only)* | The OData description of a payload. |
```

Comment:
Remove this from the document (this is the result of a bug in the documentation generator tool).  That issue is being worked, but for now, we'll manually remove this from the generated text.